examples/connect/beta-migration/values-base.yaml

```diff
@@ -0,0 +1,202 @@
+launcher:
```

Comment:
Why don't we set the config bits at the top, leave the `jammy` images as the "default" from the chart, and add the `additional` images as the bionic ones that are `exact` matches? Wouldn't that simplify this file a bit?

Reply:
Good call. Done!